src/components/AddToCart/AddToCart.js

```diff
@@ -1,10 +1,33 @@
+import {nanoid} from 'nanoid';
 import styled from 'styled-components';
 
+import useStore from '../../hooks/useStore.js';
+
 export default function AddToCart({currentMenu}) {
+  const {shoppingItems, addShoppingItem} = useStore(state => ({
+    shoppingItems: state.shoppingItems,
+    addShoppingItem: state.addShoppingItem,
+  }));
+
+  function handleAddToCart(currentMenu) {
+    const findItem = shoppingItems.find(item => item.name === currentMenu.name);
+    if (!findItem) {
+      const newShoppingItem = {
+        id: nanoid(),
+        name: currentMenu.name,
+        image: currentMenu.image,
+        price: currentMenu.price,
+        quantity: 1,
+      };
+      addShoppingItem(newShoppingItem);
+    }
+    alert('Added to cart');
```

Comment:
Maybe for a later user story it would be nice to replace the `alert` with react toastify. https://github.com/fkhadra/react-toastify
It is a rather simple to use library for messages :)

Reply:
Yes,I'll do it in next user story. Thanks for the information, it's really helpful.